modules/pubsub/registry.go

```diff
@@ -52,7 +52,11 @@ func (r *Registry) FindPublisher(path string) IPublisher {
 	r.publock.RLock()
 	defer r.publock.RUnlock()
 
-	return r.Publishers[path]
+	pub, ok := r.Publishers[path]
+	if ok {
+		return pub
+	}
+	return nil
```

Comment:
Why is that the "pokemon bug"? Is it better because if the key is not in the map it returns the "zero value" of the type, and the zero value of an interface is nil anyway, isn't it?

Reply:
Hm, that may have been an oversight on my part. I assumed that the map was pointing at structs, in which case this function would have created new structs for unknown keys, but it should be fine for interfaces.